Formula/xcode-kotlin.rb

```diff
@@ -0,0 +1,56 @@
+class XcodeKotlin < Formula
+  desc "Kotlin Native Xcode Plugin"
+  homepage "https://github.com/touchlab/xcode-kotlin"
+  url "https://github.com/touchlab/xcode-kotlin.git", tag: "1.2.0", revision: "3af7e5d921c3756a3adc83ad504ac0db2a5643e8"
+  license "Apache-2.0"
+  head "https://github.com/touchlab/xcode-kotlin.git", branch: "main"
+  depends_on "openjdk" => :build
+  depends_on xcode: ["12.5", :build]
+  depends_on :macos
+
+  def install
+    ohai "Deciding which architecture to build"
+    arch = `uname -m`.strip
+
+    case arch
+    when "arm64"
+      build_task = "linkReleaseExecutableMacosArm64"
+      build_dir = "macosArm64"
+    when "x86_64"
+      build_task = "linkReleaseExecutableMacosX64"
+      build_dir = "macosX64"
+    else
+      odie "Unsupported macOS architecture #{arch}."
+    end
+    ohai "Building executable for #{arch}"
+    build_status = run_shell "./gradlew --no-daemon #{build_task} preparePlugin"
+    odie "Build FAILED" unless build_status.success?
+    bin.install "build/bin/#{build_dir}/releaseExecutable/xcode-kotlin.kexe" => "xcode-kotlin"
+    share.install Dir["build/share/*"]
+  end
+
+  test do
+    run_shell "xcode-kotlin info"
```

Comment:
```suggestion
    assert_match "something" shell_output(bin/"xcode-kotlin info")
```

Reply:
Since the output from `xcode-kotlin info` is dynamic, should I add another command (e.g. version) that would output a known static string that could use the assert?

Reply:
Is there nothing static about the output?

Reply:
This is an example output of the `info` command. There's some static parts, but overall it's mostly dynamic (except for the "Bundled plugin version: x.y.z".
```
❯ xcode-kotlin info
Installed plugin version:	1.2.0
Bundled plugin version:		1.2.0

Language spec installed: Yes
LLDB init installed: Yes
LLDB Xcode init sources main LLDB init: Yes

Installed Xcode versions:
✔	Xcode 13.3.1 (13E500)	BD431FF2-C78B-4953-A3A9-0E42593C2D32	/Applications/Xcode-13.3.1.app
✔	Xcode 13.3 (13E112a) 	BD431FF2-C78B-4953-A3A9-0E42593C2D32	/Applications/Xcode13.3.app
✔	Xcode 13.1 (13A1030b)	8BAA96B4-5225-471B-B124-D32A349B8106	/Applications/Xcode13.1.app
✔	Xcode 13.0 (13A229)  	8BAA96B4-5225-471B-B124-D32A349B8106	/Applications/Xcode-13.0.0.app

✔ - plugin is compatible, x - plugin is not compatible
Run 'xcode-kotlin sync' to add compatibility for all listed Xcode versions.
```

Reply:
Only the xcode versions are problematic to check. Everything else seems fairly easy to assert.

Reply:
Anything but the `Bundled plugin version` depends on the machine it's installed to.

Reply:
Or do you mean it should check something like: `Language spec installed: (Yes|No)` (with proper regex syntax of course)?

Reply:
Since we have a pretty strict control over what's installed on CI. I think you can just base it on that.

Reply:
The `test` isn't being run on users machine?